src/components/widgets/Uenigboks.jsx

```diff
@@ -6,14 +6,14 @@ import { useFocusRef } from '../../hooks/useFocusRef';
 import './Uenigboks.css';
 import { useUenighet } from '../../hooks/useUenighet';
 
-const errorMessage = 'Du må oppgi årsak';
+const errorMessage = 'Du bør skrive inn årsak';
 
 const Uenigboks = ({ label }) => {
     const id = label + decodeURIComponent(window.location.pathname);
 
     const uenighet = useUenighet(id);
-    const [checked, setChecked] = useState(!!uenighet.value);
-    const [inputValue, setInputValue] = useState(uenighet.value || '');
+    const [checked, setChecked] = useState(uenighet.value ?? false);
+    const [inputValue, setInputValue] = useState(uenighet.value ?? '');
```

Comment:
Gir det mening å initialisere `uenighet.value` til `''` når man krysser av boksen, og dermed kunne slippe å ha default-verdier her og i `setInputValue`-kallet lengre ned?

Reply:
Fjerner man default-verdiene får man en litt kjipt visuell bug hvor avkryssningen 'flikrer' av/på når komponenten mountes.

Reply:
Default-verdiene er fine å ha for at inputfeltet skal vise riktig verdi ved mount om det finnes en uenighet der fra før.

Reply:
Ok, jeg klarer ikke å gjenskape det - det vil si jeg klarer ikke å forverre den flikringen jeg allerede ser :p Jeg ville tro det var mulig å sørge for at `uenighet.value` _aldri_ var `undefined`, men det er kanskje ikke så enkelt i praksis.